datacube/scripts/search_tool.py

```diff
@@ -9,6 +9,7 @@
 
 import csv
```

Comment:
Does anyone else dislike the naming of this module `scripts`. IMO it should be `cli`.

Maybe it doesn't matter, but I think of scripts as being quite different to a fully fledged and essential command line tool...